doc/introduction/measurements.rst

```diff
@@ -8,23 +8,21 @@ Measurements
 
 .. currentmodule:: pennylane.measure
 
-PennyLane can extract different types of measurement results: The expectation of an observable
-over multiple measurements, its variance, or a sample of a single measurement.
+PennyLane can extract different types of measurement results: the expectation of an observable
+over multiple measurements, its variance, or samples of a single measurement.
 
 For example, the quantum function shown in the previous section
-used the :func:`expval <pennylane.expval>` measurement:
+used the :func:`~pennylane.expval` measurement:
 
 .. code-block:: python
 
-    import pennylane as qml
-
     def my_quantum_function(x, y):
         qml.RZ(x, wires=0)
-        qml.CNOT(wires=[0,1])
+        qml.CNOT(wires=[0, 1])
         qml.RY(y, wires=1)
         return qml.expval(qml.PauliZ(1))
 
-The three measurement functions can be found here:
+For more details on the available measurement functions:
```

Comment:
This feels a bit weird. First off, there's no verb (i.e., check out the following links). But more importantly, there isn't really any content on those pages (just one-liner docstrings). Suggest rewriting this part

Reply:
Changed to:
> The available measurement functions are:
Future-proofs us in case we add more (i.e., covariance) :)